### PR TITLE
Add more zh_CN translate

### PR DESCRIPTION
**Pull request by huacnlee**

### CODE REVIEW

**Comment by huacnlee** [comment]
Can I be responsible for Chinese reviewer? Because this first version of translate

**Comment by ecnelises** [comment]
I can be the Chinese reviewer 😄 But I see some parts in the pages are hard-coded English text in template. Will it be necessary to add them with locale?

File: config/locales/zh-CN.yml [hunk 1]
```diff
@@ -22,8 +22,8 @@ zh-CN:
   activerecord:
     attributes:
       linkset:
-        bugs:
-        changelog:
+        bugs: Bugs
+        changelog: Changelog
```
Comment (sonalkr132) [inline config/locales/zh-CN.yml]: Yeah, this is not needed. It will fall back to english words by default.

Comment (ecnelises) [reply config/locales/zh-CN.yml]: Bugs - '漏洞'

Changelog - '更新日志'

**Comment by sonalkr132** [comment]
superseded by #1803

**Pull request closed without merging**